Try to preserve original formatting introduced by the Helm (#28)
giantswarm/app-catalog-cleanup-tool: CHANGELOG.md

### Fixed

- Do not try to remove non-existing app

> - Try to preserve the original formatting introduced by the `helm repo index ...`

Reviewer: ```suggestion
- Try to preserve standard formatting of the helm index.yaml.
```

Author: done, thx!
